packages/material-ui/src/Modal/ModalManager.test.js

```diff
@@ -9,6 +9,15 @@ describe('ModalManager', () => {
   before(() => {
     modalManager = new ModalManager();
     container1 = document.createElement('div');
+    container1.style.padding = '20px';
+    Object.defineProperty(container1, 'scrollHeight', {
+      value: 100,
+      writable: false,
```

Comment:
~This will not really prevent writing. All it does is throw  an error. I would rather `sinon.stub` this with a fake setter:~

`sinon.stub(container1, 'scrollHeight').get(() => 100).set(() => {})`

`writable: false` will actually not trigger an error. Would've been nice if sinon would support this out of the box.